exec-c-api/src/capi_metering.rs

```diff
@@ -52,8 +52,14 @@ pub unsafe extern "C" fn vm_exec_instance_set_points_limit(
     limit: u64,
 ) -> vm_exec_result_t {
     let capi_instance = cast_input_const_ptr!(instance_ptr, CapiInstance, "instance ptr is null");
-    capi_instance.content.set_points_limit(limit);
-    vm_exec_result_t::VM_EXEC_OK
+    let result = capi_instance.content.set_points_limit(limit);
+    match result {
```

Comment:
We could also turn this into a macro, the way we did with `cast_input_const_ptr!` above.

Reply:
will address in next pr